src/Component/Filter/FilterTree/FilterTree.tsx

```diff
@@ -368,20 +368,12 @@ export const FilterTree: React.FC<FilterTreeProps> = ({
     onFilterChange(newFilter);
   };
 
-  /**
-   * Expand handler which is passed to the Tree.
-   * Sets the expandedKeys to the state and so back to the tree.
-   */
-  const onExpand = (newExpandedKeys: (number|string)[]) => {
-    setExpandedKeys(newExpandedKeys as string[]);
-  };
-
   return (
     <Tree
       className="gs-filter-tree"
       draggable={true}
-      expandedKeys={expandedKeys}
-      onExpand={onExpand}
+      defaultExpandAll={true}
+      {...passThroughProps}
```

Comment:
Were the `passThroughProps` added above the `onDrop` on purpose?

Reply:
Yes the `onDrop` behaviour should not be overwritten.